kw

```diff
@@ -36,6 +36,18 @@ KW_STATE_DIR="${XDG_STATE_HOME:-"${HOME}/.local/state"}/${KWORKFLOW}"
 # ENV dir name
 ENV_DIR='envs'
 
+# Export external variables required by kworkflow
+export KWORKFLOW
+
+#INJECT_CODE_TRACING_SETUP
+
+# This is the one and only time a file will be sourced this way.
+# The include function (sourced from this file) should always be used for file sourcing.
+. "${KW_LIB_DIR}/lib/kw_include.sh" --source-only
+
+# Dev mode has to begin after sourcing lib/kw_include.sh
+# Otherwise, it is not possible to use helpers.
+
 ##BEGIN-DEV-MODE##
```

Comment:
I had an offline conversation with @rodrigosiqueira and he mentioned changing the name DEV-MODE to something else like REPO-MODE, or else (I like REPO-MODE, but feel free to say your piece about other names 👍). So see if you can add a commit previous to this one just to change the name.

Reply:
That is a better name indeed. I'll change it.